runtime_lib/ProfilingLib.c

```diff
@@ -5,12 +5,12 @@
 
 #include "Utils.h"
 
-static long long opcodecount[OPCODE_CYCLE_ARRAY_LEN] = {0};
+static long long unsigned opcodecount[OPCODE_CYCLE_ARRAY_LEN] = {0};
 
 void doProfiling(int opcode) {
   assert(opcodecount[opcode] >= 0 && 
          "dynamic instruction number too large to be handled by llfi");
-  opcodecount[opcode]++;
+  opcodecount[opcode]++; 
```

Comment:
Please remove the trailing whitespaces.

Reply:
Made changes in the commit: [097935c](https://github.com/DependableSystemsLab/LLTFI/pull/18/commits/097935c54300329580a27712ec07b0d2cdb294a3)